columnphysics/icepack_therm_itd.F90

```diff
@@ -992,19 +984,21 @@ subroutine lateral_melt (dt,         ncat,       &
 
       character(len=*), parameter :: subname='(lateral_melt)'
 
-      flag = .false.
+      if (tr_fsd) d_afsd_latm = c0
+
+      if (.not. any(rsiden(:) > c0)) return  ! no lateral melt, get out now
+
+!      write(warnstr,*) 'LM ',rsiden(1)
+!      call icepack_warnings_add(warnstr)
+
       dfhocn   = c0
       dfpond   = c0
       dfresh   = c0
       dfsalt   = c0
       dvssl    = c0
       dvint    = c0
-      bin1_arealoss  = c0
-      tmp  = c0
-      vicen_init = c0
-      G_radialn  = c0
-      delta_an   = c0
-      rsiden     = c0
+      vicen_init(:) = vicen(:)
+      vsnon_init(:) = vsnon(:)
```

Comment:
If these _init values are not the values at the beginning of the timestep, then I suggest naming them something else to avoid confusion, like vicen0.

Reply:
The _init notation is okay (we aren't consistent about this in other parts of the code).  I edited the PR language to be more clear.